adhocracy-plus/assets/scss/_form.scss

```diff
@@ -77,7 +77,6 @@ div.cke_focus {
 .form-check,
 .form-group {
     .django-ckeditor-widget,
-    .btn:not([type="radio"], [type="checkbox"]),
```

Comment:
ahh that's annoying, it did break the styling in multiple places however, did you test when reviewing?

Reply:
yes, the issue was there again. maybe then we make it only affect for that particular case?

Reply:
@phillimorland as this will revert a bugfix, I added another commit to fix just that particular case. Should not break in other cases. can you please test and merge in case it is okay?